src/vaadin-text-field-mixin.html

```diff
@@ -571,6 +601,9 @@
       this.shadowRoot.querySelector('[name="input"], [name="textarea"]')
         .addEventListener('slotchange', this._onSlotChange.bind(this));
 
+      this._onHelperSlotChange();
```

Comment:
Can this be tested?

Reply:
Tested here: [1](https://github.com/vaadin/vaadin-text-field/blob/4a20bf4aa9f84b881367d33740bf14a4bf7bc36e/test/text-field.html#L358).

The test fails with shady DOM if this line is removed.

Reply:
Done